doc/mgr/nfs.rst

```diff
@@ -1,71 +1,31 @@
 .. _mgr-nfs:
 
-
 =============================
 CephFS & RGW Exports over NFS
 =============================
 
-CephFS namespaces and RGW buckets can be exported over NFS protocol using the
-`NFS-Ganesha NFS server`_
-
-Requirements
-============
-
--  Latest Ceph file system with mgr enabled
--  ``nfs-ganesha``, ``nfs-ganesha-ceph``, ``nfs-ganesha-rados-grace`` and
-   ``nfs-ganesha-rados-urls`` packages (version 3.3 and above)
+CephFS namespaces and RGW buckets can be exported over NFS protocol
+using the `NFS-Ganesha NFS server`_.
 
-.. note:: From Pacific, the nfs mgr module must be enabled prior to use.
+The ``nfs`` manager module provides a general interface for managing
+NFS exports of either CephFS directories or RGW buckets.  Exports can
+be managed either via the CLI via the ``ceph nfs export ...`` commands
+or via the dashboard.
 
-Ganesha Configuration Hierarchy
-===============================
+The deployment of the nfs-ganesha daemons can also be managed
+automatically if either the :ref:`cephadm` or :ref:`mgr-rook`
+orchestrators are enabled.  If neither are in use (e.g., Ceph is
+deployed via an external orchestrator like Ansible or Puppet), the
+nfs-ganesha daemons must be manually deployed; for more information,
+see :ref:`nfs-ganesha-config`.
 
-Cephadm and rook starts nfs-ganesha daemon with `bootstrap configuration`
-containing minimal ganesha configuration, creates empty rados `common config`
-object in `.nfs` pool and watches this config object. The `mgr/nfs`
-module adds rados export object urls to the common config object. If cluster
-config is set, it creates `user config` object containing custom ganesha
-configuration and adds it url to common config object.
+.. note:: Starting with Ceph Pacific, the ``nfs`` mgr module must be enabled.
```

Comment:
Mention about dashboard and cli incompatibility

Reply:
Mention what?  Current master will convert dashboard exports to mgr/nfs style.  The intention is to update dashboard to use mgr/nfs before this is backported for the next pacific point release, at which point the dashboard and cli will work together.

Reply:
Okay, I thought the dashboard compatibility work will take longer.